src/components/UtilityNavItem/UtilityNavItem.tsx

```diff
@@ -67,28 +88,32 @@ export const UtilityNavItem = React.forwardRef<HTMLLIElement, Props>(
       }
     }
 
-    const componentClassName = clsx(
-      'utility-nav__item',
-      className,
-      isActive && 'eds-is-active',
-    );
+    const componentClassName = clsx(styles['utility-nav__item'], className, {
+      [styles['eds-is-active']]: isActive === true,
+    });
 
     return (
       <li className={componentClassName} ref={utilityNavItemRef} {...other}>
         <TagName
           className={styles['utility-nav__link']}
           href={href}
-          onClick={togglePanel}
+          onClick={togglePopover}
           aria-expanded={!href && children ? isActive : undefined}
         >
-          <span className={styles['utility-nav__text']}>{text}</span>
-          {children && (
-            <Icon
-              purpose="decorative"
-              name="expand-more"
-              className={styles['utility-nav__icon']}
-            />
+          {itemBefore && (
+            <div className={styles['utility-nav__item-before']}>
+              {itemBefore}
+            </div>
           )}
+          <span
+            className={
+              hideText
+                ? `${styles['utility-nav__text']} ${utilityStyles['u-is-vishidden']}`
+                : styles['utility-nav__text']
+            }
+          >
+            {text}
+          </span>
```

Comment:
Currently we prefer using an `aria-label` over `hideText`. We're looking into the i18n a11y concerns Brad brought up as well, but this at least keeps it consistent

Reply:
Happy to do whatever makes you all feel good as long as it's consistent. Utility classes are something I'm curious about in terms of CSS modules. Seems like we'd want those to be global styles you can pass in throughout the library and people can use them without having to import them. I'll admit I'm a little less experienced with CSS modules so still learning the nuance

Reply:
@anniehu4 Do you want this changed to aria label or should we leave it for now until we find a more permanent solution? Happy to do either. Thanks!

Reply:
> Utility classes are something I'm curious about in terms of CSS modules. Seems like we'd want those to be global styles you can pass in throughout the library and people can use them without having to import them. I'll admit I'm a little less experienced with CSS modules so still learning the nuance

Yeah, that makes sense to me. I'm thinking we could export those as regular `.css` files and then import at a top-level (e.g. `preview.js` for storybook, `index.js` for the App) so they're available globally